README.adoc

```diff
@@ -3,39 +3,19 @@
 image::logo.png[riff logo, link=https://projectriff.io/]
 = A FaaS for Kubernetes
 
-The riff project builds on top of the link:https://github.com/knative/[Knative] project's build, serving and eventing features.
+The riff CLI helps developers build and run functions using link:https://github.com/knative/docs[Knative].
+It includes commands to install Knative in a Kubernetes cluster, and for managing functions, services, channels, and subscriptions.
 
 == Installation of the latest release
 
-Official binary release are available from the link:https://github.com/projectriff/riff/releases[Releases] page.
-
-See link:https://projectriff.io/docs/getting-started-with-knative-riff-on-minikube/[Getting started on Minikube] or
+Official binary release are available from the link:https://github.com/projectriff/riff/releases[Releases] page. See link:https://projectriff.io/docs/getting-started-with-knative-riff-on-minikube/[Getting started on Minikube] or
 link:https://projectriff.io/docs/getting-started-with-knative-riff-on-gke/[Getting started on GKE] for how to install the riff CLI and the riff system.
 
-=== Community releases
-
-NOTE: Community releases are not created, managed or monitored by the riff team.
-
-To install the `riff` CLI for MacOS with Homebrew:
-
-[source, bash]
-----
-brew install starkandwayne/cf/riff
-----
-
-To install the CLI for Debian/Ubuntu Linux:
-
-[source, bash]
-----
-wget -q -O - https://raw.githubusercontent.com/starkandwayne/homebrew-cf/master/public.key | apt-key add -
-echo "deb http://apt.starkandwayne.com stable main" | tee /etc/apt/sources.list.d/starkandwayne.list
-apt-get update
-apt-get install riff
-----
+See link:Community-Releases.adoc[Community releases] for Mac Homebrew and Debian Apt packages.
```

Comment:
I'm not convinced these will continue to be supported.

@drnic care to comment?

Reply:
@drnic Great stuff, thanks. We'll keep these instructions around until such time as we replace them with our own facilities.